pkg/services/accesscontrol/evaluator/evaluator.go

```diff
@@ -5,13 +5,18 @@ import (
 
 	"github.com/gobwas/glob"
 
+	"github.com/grafana/grafana/pkg/infra/metrics"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 // Evaluate evaluates access to the given resource, using provided AccessControl instance.
 // Scopes are evaluated with an `OR` relationship.
 func Evaluate(ctx context.Context, ac accesscontrol.AccessControl, user *models.SignedInUser, action string, scope ...string) (bool, error) {
+	timer := prometheus.NewTimer(metrics.MAccessEvaluationsSummary)
+	defer timer.ObserveDuration()
+	metrics.MAccessEvaluationCount.Inc()
```

Comment:
This is technically the same as `access_evaluation_duration_count`, since a Prometheus histogram, but I don't think there's any harm in having it separately as well. It's somewhat clearer.